ledger/src/blockstore_db.rs

```diff
@@ -1227,6 +1245,39 @@ impl TypedColumn for columns::OptimisticSlots {
     type Type = blockstore_meta::OptimisticSlotMetaVersioned;
 }
 
+impl Column for columns::MerkleRootMeta {
+    type Index = (Slot, u64);
+
+    fn index(key: &[u8]) -> (Slot, u64) {
+        let slot = BigEndian::read_u64(&key[..8]);
+        let set_index = BigEndian::read_u64(&key[8..]);
```

Comment:
what is the reason we use big-endian?
aren't almost all machines little-endian these days?

Reply:
I'm not actually sure, I just followed the convention set by the other columns which use `(Slot, fec_set_index)` as a key

Reply:
@steviez @yhchiang-sol
do you guys have any context why the keys are encoded with `BigEndian`?

Reply:
I'm actually not positive on why `BigEndian`; it has been like that for as long as I can remember. Will dig through `git` in a little to see if I can uncover anything

> and do we have to follow same convention for new columns?

From a technical point of view, I don't know of any reason that they "have to" be `BigEndian`. That being said, I'd prefer consistency here. If there was a compelling reason to switch to `LittleEndian`, I think I'd still push for this change to use `BigEndian` to be consistent with the other columns, and then have a later change shift all the columns over to `LittleEndian`

Reply:
> do you guys have any context why the keys are encoded with BigEndian?

From RocksDB's perspective, it doesn't require any key-order, and the user (i.e. us) will have to specify the correct comparator that works with their key-format.

> and do we have to follow same convention for new columns?

I would like to second @steviez's opinion.  While it isn't required to have all column families using the same format in RocksDB, it's good to have all the slot-based column families following the same format in our use case.    In addition to the code consistency benefit, it also makes operations like multi-get across multiple column-families using the same key-prefix or key easier.

Reply:
Looks like the use of `BigEndian` goes very far back to https://github.com/solana-labs/solana/pull/2897